bench/wtperf/wtperf.c

```diff
@@ -753,14 +785,18 @@ run_mix_schedule(CONFIG *cfg, WORKLOAD *workp)
 	memset(workp->ops, WORKER_READ, sizeof(workp->ops));
 
 	pct = (workp->insert * 100) /
-	    (workp->insert + workp->read + workp->update);
+	    (workp->insert + workp->read + workp->truncate + workp->update);
```

Comment:
If you setup a worker thread to do both truncate and other operations, won't the sleeps in the truncate code cause the workload to behave strangely?